sky/cli.py

```diff
@@ -772,6 +772,7 @@ def _make_task_or_dag_from_entrypoint_with_overrides(
     else:
         task = sky.Task(name='sky-cmd', run=entrypoint)
         task.set_resources({sky.Resources()})
+        task.update_envs(env)
```

Comment:
could we add a comment saying that for dag yamls, we do env override in the `dag_utils.load_chain_dag_from_yaml` function? I was a little bit confused when I only looked at code updates within this file and saw we changed from updating the env vars for all cases to only updating sky-cmd cases..